app/_features/event/components/event-status.tsx

```diff
@@ -0,0 +1,16 @@
+import { Code } from '@nextui-org/react';
+
+export function StatusPublished() {
+  return (
+    <Code className="rounded-sm bg-emerald-950 text-xs text-emerald-300 ring-1 ring-emerald-800">
+      Published
+    </Code>
+  );
+}
+export function StatusDraft() {
+  return (
+    <Code className="rounded-sm bg-blue-950 text-xs text-blue-300 ring-1 ring-blue-800">
+      Draft
+    </Code>
+  );
```

Comment:
didn't know that there's actually html semantic for code, is there any trade of using chip instead of `<div>` or `<span>` apart from the simplicity, just curious

Reply:
Yes, semantic HTML introduces the meaning of each element rather than the presentation only. `<code>` element is used for representing the code-based text. It's not supposedly used for label or item's status.

There is no difference of using `<chip>` component or using `<div>` and `<span>` directly. The chip component is just a component which mostly `<div>` and `<span>` based if you inspect the code. Between using the `<div>` and `<chip>`, you still need to style those to adjust the visual to make it similar to the design. In this case, I will rather create the visual from scratch using native element like div or span.